scripts/pylint_custom_plugin/tests/test_pylint_custom_plugins.py

```diff
@@ -2677,3 +2677,56 @@ def test_disable_pylint_alias(self):
 
         with self.assertNoMessages():
             self.checker.visit_module(node)
+
+class TestCheckEnum(pylint.testutils.CheckerTestCase):
```

Comment:
Similar comment to the one above, I think we should add a test that shows both messages if both pylint checks fail in the same enum class.